src/librustc_incremental/calculate_svh/mod.rs

```diff
@@ -58,6 +58,7 @@ impl<'a, 'tcx> SvhCalculate for TyCtxt<'a, 'tcx, 'tcx> {
         {
             let mut visit = StrictVersionHashVisitor::new(&mut state, self);
             krate.visit_all_items(&mut visit);
+            krate.visit_all_items(&mut IdVisitor::new(&mut visit));
```

Comment:
We probably don't need to do this for the SVH since we are visiting all `use` declarations too in that case, right? Although, that would kind of rely on things being visited in lexical order...